### ISSUE TITLE
unexpected behaviour of validation when using createSubmitHandler

### ISSUE THREAD

**Issue by mbornath** [pablo-abc/felte#38]
## Description

For a form we have to provide two different submit actions that perform different tasks and also differ in validation. To achieve that, we use the `createSubmitHandler` helper.

Please have a look at this simplified reproduction for further insight: https://svelte.dev/repl/b9f7503bbc5b4e838fcbe2bce4e9bbd9?version=3.34.0

You will see a form with two simple text inputs and two submit buttons. The `standard` submit validates only the first input field. The `second` submit uses the same validation but additionally should check the validity of the second input field.

## Expected Behavior

if first text input has no value:
- do not perform submit for `standard` submit or `second` submit
- give visual error feedback in UI for first text input

if first input has a value and second input has no value:
- perform submit for `standard` submit but not for `second` submit
- give visual error feedback in UI for second text input when using `second` submit

## Actual Behavior

if first text input has no value:
- does not perform submit for `standard` submit or `second` submit
- gives visual error feedback in UI for first text input

if first input has a value and second input has no value:
- performs submit for `standard` submit but not for `second` submit
- no visual error feedback in UI for second text input when using `second` submit

## Possible Solution

Instead of just returning the appropriate `errors` object in the `second` validation handler, additionally i can use `setError` to achieve our goals.

```js
	const secondSubmit = createSubmitHandler({
		validate: (values) => {		
			const errors = {}

			if (!values.second) {
				errors.second = 'required'
                                 setError('second', 'required')
			}

			return errors
		},
		onSubmit: (values) => {
			console.log('second submit', values);
		}
	})

```

Maybe I got something wrong , but i would expect `felte` to handle that case.

Thanks for your efforts.

**Comment by pablo-abc** [comment]
Just pushed a fix for this at `felte@0.7.12`, thanks for opening this issue! 

Your example in the REPL should be working as expected now as well. Feel free to close this if you don't find any further issues.

**Comment by mbornath** [comment]
Nice. Thank you very much
